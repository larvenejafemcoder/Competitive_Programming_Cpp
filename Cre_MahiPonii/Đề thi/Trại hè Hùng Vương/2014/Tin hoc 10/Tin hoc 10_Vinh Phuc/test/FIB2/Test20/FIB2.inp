26
13 265
24 54958
45 1732254023
37 32684508
26 43631
2 1
41 28539738
4 4
41 242325940
4 4
5 8
22 13021
28 220268
30 772520
9 12
7 3
37 5320516
20 3132
43 110772987
6 4
24 70264
28 181211
18 3532
37 9291389
17 1012
2 2
