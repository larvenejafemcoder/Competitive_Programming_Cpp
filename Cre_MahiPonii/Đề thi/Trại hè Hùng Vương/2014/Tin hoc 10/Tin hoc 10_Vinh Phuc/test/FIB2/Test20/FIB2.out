101
20992
661662160
12484371
16666
1
10901210
2
92560273
2
3
4974
84135
295077
5
1
2032256
1196
42311516
2
26839
69217
1349
3548995
387
1
